adam/batch.py

```diff
@@ -111,10 +111,24 @@ def __init__(self, params):
         """
         Param options are:
 
-            --- epoch and state_vector are required! ---
+            --- epoch is required! ---
             epoch (str): the epoch associated with the state vector (IS0-8601 format)
+
+            --- either state_vector or keplerian_elements is required! ---
+            --- note that if keplerian_elements are provided, state_vector will be ignored
+            --- by server side even if also provided. ---
             state_vector (list): an array with 6 elements [rx, ry, rz, vx, vy, vz]
-                representing the position and velocity vector of the object.
+                representing the cartesian coordinates (the position and velocity vector)
+                of the object.
+            keplerian_elements (list): an array with 7 elements representing the
+                keplerian coordinates of the object. The elements are:
+                    0: semi_major_axis (float): Semimajor axis (km)
```

Comment:
I agree that people will probably copy-paste, but all these values are floats, and a bunch of them have the same units (degrees), so it's very easy to be off-by-one. If not a dictionary, maybe at least a named tuple?